### DISCUSSION TITLE
Online Workshop - Introduction to MDAnalysis and Molecular Nodes

### DISCUSSION THREAD

**Discussion by jennaswa** [MDAnalysis/mdanalysis#4448] [Announcements]
Do you know some Python but are still a beginner to MDAnalysis and want to learn about analyzing and visualizing molecular simulation data? Join us Feb 28 at 3:00 UTC for a free online workshop on a basic workflow with MDAnalysis and Molecular Nodes!

Spots are limited, so make sure to apply before Feb 19: https://www.mdanalysis.org/2024/02/05/mdaMNworkshop/

[Flyer - February 2024 MDA_MN Workshop.pdf](https://github.com/MDAnalysis/mdanalysis/files/14168508/Flyer.-.February.2024.MDA_MN.Workshop.pdf)

**Comment by orbeckst** [comment]
See as a simple example this simulation of the transition of the enzyme AdK from its closed to its open conformation (the trajectory is part of the MDAnalysis test data — `MDAnalysisTests.datafiles.PSF` and `MDAnalysisTests.datafiles.DCD`) rendered with [MolecularNodes](https://bradyajohnston.github.io/MolecularNodes/) in Blender by @bradyajohnston (click on the image below to go to YouTube and see the video).

[![AdK transition rendered with MolecularNodes](https://img.youtube.com/vi/YhqGtnfqN48/maxresdefault.jpg)](https://www.youtube.com/watch?v=YhqGtnfqN48)

**Comment by SatyaPrakashPalai** [comment]
Hello @jennaswa,
    I just came across this repository, and it's fascinating. I have been using LAMMPS for molecular dynamics and have experience in Python. I know the deadline for grabbing a seat for the workshop is over, but I wonder if I could still attend. It would help me increase my knowledge. 
Thank you

**Reply by jennaswa** [reply]
Thank you, @SatyaPrakashPalai,  for your interest in the workshop. Although the application/participation slots are now full, the workshop will be recorded and made publicly available on the [MDAnalysis YouTube channel](https://www.youtube.com/@mdanalysis3040) shortly after the workshop has taken place. You may also check out the recording from an [October 2023 MDAnalysis workshop](https://www.youtube.com/watch?v=njzoNzOwR78) and this tutorial on [getting started in Molecular Nodes](https://www.youtube.com/watch?v=CvmFaRVmZRU), which contain similar material to this current workshop.

Please let us know if you have any further questions.
